app/actions/games.js

```diff
@@ -0,0 +1,21 @@
+import { Config }       from '../../config';
+import { API }          from '../utils/api';
+import { checkVersion } from './utils';
+
+export const SET_GAMES  = 'SET_GAMES';
```

Comment:
nit: whitespace

also, can you rename this file to be `game.js`? the action files are all singular